Common/Statistics/PortfolioStatistics.cs

```diff
@@ -28,6 +28,7 @@ namespace QuantConnect.Statistics
     public class PortfolioStatistics
     {
         private const decimal RiskFreeRate = 0;
+        private const decimal MaximumTradeableVolume = 0.025m;
```

Comment:
`MaximumTradeableVolume`? or is this more like a `MaximumTradeableVolumePercent` ? This concerns me since it looks like a _magic_ number and there's no description as to why it's magic and as to how the value was determined. I've been reading these factors as 2.5%